users/service.go

```diff
@@ -575,9 +589,9 @@ func (svc service) OAuthCallback(ctx context.Context, state mgoauth2.State, clie
 		rclient, err := svc.RegisterClient(ctx, "", client)
 		if err != nil {
 			if errors.Contains(err, repoerr.ErrConflict) {
-				return &magistrala.Token{}, errors.New("user already exists")
+				return &magistrala.Token{}, errors.Wrap(svcerr.ErrConflict, err)
```

Comment:
I don't think we need to wrap with `svcerr.ErrConflict` since it contains `repoerr.ErrConflict` and they have the same error message

Reply:
I don't understand, `repoerr.ErrConflict` has message `entity already exists`.

Reply:
`svcerr.ErrConflict` and `repoerr.ErrConflict` have the same error message the difference is one should be used at service layer while the other should be used in repository